CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 5.8
+
+## Non-Breaking Changes
+
+* **Added** new property `showAllErrors` to `frost-bunsen-form` component that when true informs bunsen to show all error messages even before there is user interaction. This property defaults to `false` making it backwards compatible.
+* **Added** new property `isRequiredError` to required errors returned by the `onValidation` action/property with the value `true` to distinguish which error messages are required error messages versus which ones are not.
```

Comment:
May want to clarify what a "required error message" is, it makes it sound like some error messages are optional. What we're talking about here are error messages indicating a missing required field, it might be good to explicitly say that.